django_fsm_freeze/models.py

```diff
@@ -30,11 +32,23 @@ def freeze_check(self) -> None:
     @classmethod
     def config_check(cls) -> None:
         errors = defaultdict(list)
+        try:
+            fsm_state_field = cls._meta.get_field(cls.FSM_STATE_FIELD_NAME)
+        except FieldDoesNotExist:
+            errors[cls.FSM_STATE_FIELD_NAME].append(
+                f'{cls.FSM_STATE_FIELD_NAME!r} field does not exist.'
+            )
+        else:
+            if not isinstance(fsm_state_field, FSMField):
+                errors[cls.FSM_STATE_FIELD_NAME].append(
```

Comment:
For the error message, `errors['FSM_STATE_FIELD_NAME'].append(....)`?

Reply:
makes sense